src/content/topics/guides/best-practices/terraform.mdx

```diff
@@ -51,13 +51,14 @@ With each example, we'll discuss why Terraform was suited to that scenario. We'l
 These are the practices we recommend to avoid conflicts that can arise when some of your flags are managed through Terraform and others are managed through the LaunchDarkly dashboard:
 
 * Tag everything Terraform manages with a `managed-by-terraform` tag.
-* Use custom roles to enforce that resources and properties managed by Terraform can not be manipulated by the LaunchDarkly UI. For example, putting most team members in a role that does not have write access to the resources managed by Terraform.
-* For more complex patterns, add another tag such as `allow-ui-toggle`, to make it so rules are read-only but the toggle state is changeable.
+* Use custom roles to enforce that resources and properties managed by Terraform can not be manipulated by the LaunchDarkly UI. For example, putting most team members in a role that does not have write access to the resources managed by Terraform. 
+* Avoid, unless necessary, using Terraform to make changes to flag targeting rules or flag variations. The benefit of managing these actions in the UI is the ability to use LaunchDarkly's comment, audit log, scheduling, flag trigger, and approval workflow features.
```

Comment:
we might want to expand on the audit log part of this. Just to clarify that, it's not that the audit log won't capture the change but it won't really be attributable

Reply:
@InTheCloudDan I think that's out of scope for that the rest of this section.